scripts/swupd-performance/swupdservicesetup.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
```

Comment:
It's a bash, so rename it to a .bash. I would go with "swupd-logging-setup.sh "

Reply:
okay